src/ElasticApmAgentLaravelOctane/EventListeners/RequestReceivedHandler.php

```diff
@@ -21,8 +21,12 @@ public function handle(RequestReceived $event): void
         /** @var OctaneApmManager $manager */
         $manager = $event->app->make(OctaneApmManager::class);
 
-        $manager->beginTransaction($event->request->method() . ' ' . $this->getRouteUri($event), 'request');
-        $manager->beginAndStoreSpan('RequestResponse', 'request');
+        $routeUri = $this->getRouteUri($event);
+        if ($routeUri === "/") {
+           $manager->beginTransaction($event->request->method() . ' /', 'request');
+        } else {
+           $manager->beginTransaction($event->request->method() . ' /' . $routeUri, 'request');
+        }        
```

Comment:
It would be better for you to add this "/" prefixing logic in the "getRouteUri" method instead :smile:

Reply:
Roger that 🤟